socorro/external/postgresql/models.py

```diff
@@ -1545,10 +1545,13 @@ class MissingSymbols(DeclarativeBase):
     __tablename__ = 'missing_symbols'
 
     date_processed = Column(u'date_processed', DATE(), nullable=False)
-    debug_file = Column(u'debug_file', TEXT(), nullable=True, primary_key=True)
+    filename = Column(u'filename', TEXT(), nullable=True)
+    debug_file = Column(u'debug_file', TEXT(), nullable=False)
```

Comment:
This is going to need a rebase as I'm removing the PRIMARY KEY on this column in another PR.